shaders/commonProfileShaders.cpp

```diff
@@ -55,11 +56,17 @@ namespace GLTF
         ((std::istream*)png_get_io_ptr(pngPtr))->read((char*)data, length);
     }
 #endif
+	static std::unordered_map<std::string, bool> haveAlphas;		// Caching to remove up to 98% execution time
```

Comment:
This should probably just be a regular map rather than an unordered one. They're usually faster and I don't think being unordered is a necessity here.